src/common/config.ts

```diff
@@ -1,14 +1,28 @@
 import * as uuidv4 from 'uuid/v4';
 import { loadConfig } from 'snyk-config';
+import * as fs from 'fs';
 
 const config: Record<string, any> = loadConfig(__dirname + '/../..', {
   secretConfig: process.env.CONFIG_SECRET_FILE,
 });
 
+const namespacesFilePath = '/etc/config/namespaces';
```

Comment:
Where is Helm charts you define this localtion?

Reply:
https://github.com/snyk/kubernetes-monitor/pull/665/files#diff-4d1b7a6d5d6f40fba7fa252f965e47092c24f6c7715d9be94adc3b320b303b12R54-R57

Reply:
`snyk-monitor/templates/configmap.yaml`